docs/getting-started/09-trigger-microservice-with-event.md

```diff
@@ -7,7 +7,7 @@ This tutorial shows how to trigger the deployed `orders-service` microservice wi
 
 ## Reference
```

Comment:
In general, the getting started guide mentions triggers in many places which be changed to the new Eventing Subscriptions.

Reply:
Fixed